src/backend/joanie/tests/core/test_api_course.py

```diff
@@ -61,9 +63,9 @@ def test_api_course_read_detail_anonymous(self):
         """
         target_course_runs = factories.CourseRunFactory.create_batch(
             2,
-            start=timezone.now() - timedelta(hours=1),
-            end=timezone.now() + timedelta(hours=2),
-            enrollment_end=timezone.now() + timedelta(hours=1),
+            start=timezone.now() - timedelta(days=1),
+            end=timezone.now() + timedelta(days=2),
+            enrollment_end=timezone.now() + timedelta(days=1),
```

Comment:
Debug residue ^^ (ok ok, I was wrong, the timezone was the right one !)

Reply:
yes but no harm leaving it this way for you not to doubt anymore about that, wdyt? I had left it on purpose

Reply:
I'd like the idea to have a test which is able to detect timezone issues ?